src/components/deviceProvisioningWorkflow/deviceProvisioningWorkflow.js

```diff
@@ -317,24 +323,116 @@ class DeviceProvisioningWorkflow extends React.Component {
    * @returns Returns JSX for the form summary
    */
   renderFormSummary() {
+    if (this.state.isDeviceProvisioned) {
+      return this.renderDeviceProvisionedSummary();
+    }
+    else {
+      return (
+        <div className="form-summary-container">
+          <div className="form-summary-header">{lang.PROVISION_SUMMARY}</div>
+          <div className="form-summary-details">
+            <span className="device-cnt">{this.isInteger(this.state.numDevices) ? Math.floor(this.state.numDevices) : 0}</span>
+            {this.state.isLoading ?   lang.DEVICES_ARE_PROVISIONING : lang.DEVICES_TO_PROVISION }
+          </div>
+          <div className="form-action-btns">
+            <PcsBtn onClick={this.props.onClose} value={lang.CANCEL}/>
+            <PcsBtn className="primary"
+              disabled={!this.state.formIsValid || this.state.isLoading}
+              onClick={this.apply}
+              value={this.state.isLoading ? lang.PROVISION_LOADING : lang.APPLY} />
+          </div>
+        </div>
+      );
+    }
+  }
+
+  /**
+   * Renders the summary portion of the physical device form once a device is provisioned
+   * 
+   * @returns Returns JSX for the form summary for a provisioned device, 
+   * including device id, keys and connection strings.
+   */
+  renderDeviceProvisionedSummary()
+  {
+    const primaryConnectionString = this.createConnectionString(true);
+    const secondaryConnectionString = this.createConnectionString(false);
     return (
-      <div className="form-summary-container">
+      <div className="form-summary-container  device-summary-container">
         <div className="form-summary-header">{lang.PROVISION_SUMMARY}</div>
-        <div className="form-summary-details">
+        <div className="form-summary-details device-summary-details">
           <span className="device-cnt">{this.isInteger(this.state.numDevices) ? Math.floor(this.state.numDevices) : 0}</span>
-          {this.state.isLoading ?   lang.DEVICES_ARE_PROVISIONING : lang.DEVICES_TO_PROVISION }
+          {lang.DEVICES_PROVISIONED}
+          <img src={Apply} alt={`${Apply}`}  className="check-icon" />
+          {this.renderDeviceDetail('Device ID', this.state.provisionedDevice.Id)}
```

Comment:
You might want to use [destructuring assignment](http://es6-features.org/#ObjectMatchingShorthandNotation) on the state to reduce the having to refer to state so much here.